src/main/java/com/solr/clientwrapper/rest/EmployeeController.java

```diff
@@ -66,6 +73,76 @@ public Employee getEmployeeByAddress(@PathVariable String address) {
     private SolrClient solrClient;
```

Comment:
change name of this class from employee controller

Reply:
Employee Controller was only for testing. Removed it and also the other related classes.